src/views/self/view_weapons.cpp

```diff
@@ -198,18 +198,14 @@ namespace big
 		}
 		ImGui::PopItemWidth();
 		ImGui::SameLine();
-		if (ImGui::Button("Give Weapon"))
+		if (components::button("Give Weapon"))
```

Comment:
https://github.com/YimMenu/YimMenu/blob/cb9205e478c16e6b4686f7fbdee04d0046388c2f/src/views/self/view_weapons.cpp#L93

this is what I meant, you need to use the overload with the callback in it for the natives to work